Source/WebCore/html/shadow/mac/imageControlsMac.css

```diff
@@ -27,7 +27,7 @@
 
 div#image-controls {
     display: inline-block;
-    position: relative;
+    position: fixed;
```

Comment:
This is quite a surprising change that definitely feels like it needs additional explanation. What happens if these are inside subscrollable regions?